director/jobs/templates/jobs_list_job.html

```diff
@@ -0,0 +1,48 @@
+<div class="job has-text-black">
+  <div class="job-column job-summary">
+    <span class="job-id is-block leading-1">{% if job.id %}JOB {{job.id}}{% endif %}</span>
+    <span class="tag is-uppercase job-tag {{job.colour}}">{{job.status}}</span>
+    <div class="job-icon status is-flex">
+      <i data-feather="{{job.icon}}" class="icon flex-shrink-0"></i>
+      <span class="text truncate has-text-weight-bold">{{job.method_label}}</span>
+    </div>
+  </div>
+
+  <div class="job-column job-details flex-grow is-flex flex-col">
+    <header class="job-details-header flex-wrap is-flex mb-2">
+
+      <span class="job-info job-icon job-user is-flex">
+          <img src="https://via.placeholder.com/16" width="16" height="16" alt="" class="user-avatar job-user-image icon">
+          <span class="job-user-name is-uppercase">{{job.creator.username}}</span>
+      </span>
+
+      <span class="job-info job-icon is-flex">
+          <i data-feather="calendar" class="icon flex-shrink-0"></i>
+          <span class="job-user-name is-uppercase">{{job.created|date:"F n, Y g:i A"}}</span>
+      </span>
+
+      <span class="job-info job-icon is-flex">
+          <i data-feather="clock" class="icon flex-shrink-0"></i>
```

Comment:
`clock` icon is a little confusing given the `calendar` before it. How about an hourglass?

Reply:
That's fair - it does look like an actual time stamp rather than a total time. I'm going to use an hourglass now, but it is the fontawesome one as feather icon has no such icon.